hfile.c

```diff
@@ -1131,7 +1131,7 @@ static hFILE *hopen_unknown_scheme(const char *fname, const char *mode)
 }
 
 /* Returns the appropriate handler, or NULL if the string isn't an URL.  */
-static const struct hFILE_scheme_handler *find_scheme_handler(const char *s)
+const struct hFILE_scheme_handler *find_scheme_handler(const char *s)
```

Comment:
Can you see if there is a new api to use the htslib plugin mechanism for schemes like `azb://`.`s3://`, so we don't have to expose this function?

Reply:
There are new functions to list plugins and schemes. Explained here : https://github.com/samtools/htslib/pull/1170and https://github.com/samtools/htslib/pull/1222. Not reviewed in depth yet.